packages/vega-parser/src/parsers/guides/guide-util.js

```diff
@@ -86,3 +86,46 @@ export function extendOffset(value, offset) {
     : !isObject(value) ? { value, offset }
     : { ...value, offset: extendOffset(value.offset, offset) };
 }
+
+const DISCRETE_SCALES = new Set(['ordinal', 'band', 'point']);
+
+function domainText(scaleType, scaleName) {
+  return DISCRETE_SCALES.has(scaleType)
+    ? ` with values " + domain('${scaleName}')`
```

Comment:
How should we support localization for these?

Reply:
I believe we can use `encode` and override the properties there.

Reply:
Yes, you can override `ariaLabel` for axes. This is just a default.

Reply:
I'll move some defaults to config where it makes sense.

Reply:
I'll move some defaults to config where it makes sense.